### DISCUSSION TITLE
Load Times Slow

### DISCUSSION THREAD

**Discussion by dkmbuild** [Budibase/budibase#13214] [Help] [answered]
Is there a way to trim back the speeds and requests?
![load time](https://github.com/Budibase/budibase/assets/149087656/7dd52e1d-0679-4220-83b7-6c45c2fdadd7)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dkmbuild,

Our engineering team is actively optimising our cloud infrastructure for better performance. However, like any significant undertaking, this process will take time. Unfortunately, there are no immediate actions you can take to improve performance, barring self-hosting Budibase. We appreciate your patience and understanding as we work diligently to provide you with an optimised and efficient experience.
